js/main.js

```diff
@@ -268,6 +273,11 @@ const MM = (function () {
 	 * @param {object} [options] Optional settings for the show method.
 	 */
 	const showModule = function (module, speed, callback, options = {}) {
+		// reset lockString and prepare for set the new one
+		if (options.reset === true && options.lockString) {
```

Comment:
I'd call the option "resetLockStrings" to make the usage clearer

Reply:
sure, You want to rename it or i do it ?